clients/ui/bff/internal/integrations/httpclient/http.go

```diff
@@ -112,6 +113,10 @@ func (c *HTTPClient) GET(url string) ([]byte, error) {
 }
 
 func (c *HTTPClient) POST(url string, body io.Reader) ([]byte, error) {
+	return c.POSTWithContentType(url, body, "application/json")
+}
+
+func (c *HTTPClient) POSTWithContentType(url string, body io.Reader, contentType string) ([]byte, error) {
```

Comment:
@ppadti one last quick change: can this method be called just POST (with content type as parameters)? In that way we 100% align with http package.